invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/JobSearchResultItemLayout.js

```diff
@@ -39,6 +40,11 @@ class SearchResultItemComponent extends Component {
     console.error(e);
   };
 
+  handleSuccess = () => {
+    // Reload the page after successful action
+    window.location.reload();
```

Comment:
do we need to reload the entire page? Or maybe refresh the list only?

Reply:
I opted for the simplest implementation here because at the moment you actually need to manually refresh the page in order to see the changes. Do you have a suggestion for how to refresh just the list in a simple way? Ideally something without introducing lifecycle methods like `componentDidMount/componentDidUpdate` that I am trying to avoid to introduce.